test-cases/longevity/longevity-2TB-4days-authorization-and-tls-ssl-1dis-2nondis-nemesis.yaml

```diff
@@ -22,7 +22,6 @@ n_monitor_nodes: 1
 
 instance_type_db: 'i3en.3xlarge'
 instance_type_loader: 'c4.2xlarge'
-instance_type_monitor: 't3.large'
```

Comment:
why this is removed in the PR ?

Reply:
because it is the same as the default, that is set on aws_config.yaml